MacBry-Maciej-Bryja/SimpleTodoApp1/src/main/java/com/mac/bry/simpleTodo/utilitis/PasswordUtillity.java

```diff
@@ -0,0 +1,27 @@
+package com.mac.bry.simpleTodo.utilitis;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import javax.xml.bind.DatatypeConverter;
+
+public class PasswordUtillity {
+	   
+    public static String getHashedPassword(String passwordToHash) {
```

Comment:
Metoda nie powinna nigdy zwrócić nulla, ponieważ może to doprowadzić do SQL injection. Powinieneś rzucić wyjątkiem informującym o tym, że nie hashowanie się nie powiodło.